runtime/runtime/src/lib.rs

```diff
@@ -660,6 +661,7 @@ impl Runtime {
                 receipt_ids,
                 gas_burnt: result.gas_burnt,
                 tokens_burnt,
+                receiver_id: account_id.clone(),
```

Comment:
@evgenykuzyakov Could you confirm that since currently we disallow multiple delete actions in the receipt, `account_id == account.account_id`, correct?

Reply:
This doesn't belong here. You already have `receipt_id`

Reply:
Ok. I read the attached issue and it seems reasonable, but not required. Basically, we want to include duplicated information from `Receipt` to the `ExecutionOutcome` to be able to easier process this on Eth side.

Even though it's a waste of storage, it's still helpful because it allows to identify the shard on which the execution outcome was produced. Eventually we'll need to know the block height to identify the shard, but it's fine until we have dynamic resharding.